contribs/gmf/less/mobile.less

```diff
@@ -227,15 +227,24 @@ gmf-map {
 }
 
 .ol-control button,
-.ol-touch .ol-control button {
-  font-size: 1em;
+.ol-touch .ol-control button,
+.fa:after,
+.fa:before,
+.gmf-icon:after,
+.gmf-icon:before {
+  font-size: 2.4rem;
```

Comment:
My proposal:
![screenshot from 2016-02-18 17 53 04](https://cloud.githubusercontent.com/assets/319774/13151033/896e632e-d668-11e5-8d87-5a0db5b569ec.png)
Yours:
![screenshot from 2016-02-18 17 53 12](https://cloud.githubusercontent.com/assets/319774/13151037/8ca293da-d668-11e5-9753-ac3a3dede61d.png)

I agree, it looks better.